MAINTAINERS.yml

```diff
@@ -3156,7 +3236,8 @@ West:
   collaborators:
     - tgorochowik
     - msobkowski
-  files: []
+  files:
+    - modules/hal_ambiq/
```

Comment:
```suggestion
    - modules/hal_ambiq/
    - boards/arm/apollo*/
```

Reply:
this is already covered and should be part of the platform area in any case...

```
/scripts/get_maintainer.py path boards/arm/apollo4p_blue_kxr_evb/
Ambiq Platforms
	status: maintained
	maintainers: fkokosinski
	collaborators: tgorochowik, msobkowski
	inform:
	labels: platform: Ambiq
	description:
```